_includes/articles/payment-batching.md

```diff
@@ -127,15 +124,15 @@ transaction is being paid by you.  If you had sent separate
 transactions, any onchain relationship between the payments might be
 less apparent or even non-existent.
 
-![Screenshot of a possible transaction batch in a block explorer](img/batch-screenshot.png)
+![Screenshot of a possible transaction batch in a block explorer](/img/posts/payment-batching/batch-screenshot.png)
 
 Note that transactions belonging to particular Bitcoin services are
```

Comment:
"Note that" usually doesn't add much meaning. If the aim here is to highlight that the privacy concerns in the previous paragraph aren't as bad as the reader might assume, then I think it's better to state that explicitly. Something like:

"However, this isn't such a loss of privacy that you might assume. Transactions belonging to particular Bitcoin services ..."

Reply:
I think "Note that" adds about the same amount of meaning as "However".  :-)  I personally prefer the current phrasing and it avoids the problem of the following paragraph having a sentence that itself starts with "However,".  *However,* I fine with it either way.

Reply:
Note that these are merely style suggestions. If you prefer the current phrasing, we should keep it as is.